syntax = "proto3";

package expcmake;

import "proto/address.proto";

message NameQuerry {
    string name = 1;
}

service AddressBook {
    rpc GetAddress(NameQuerry) returns (Address) {}
}